tests/phpunit/tests/customize/nav-menus.php

```diff
@@ -953,7 +953,12 @@ function test_filter_wp_nav_menu() {
 			'items_wrap'  => '<ul id="%1$s" class="%2$s">%3$s</ul>',
 		);
 
-		// Add global namespace prefix to check #41488.
+		// Add global namespace prefix to check https://core.trac.wordpress.org/ticket/41488.
+		if ( version_compare( PHP_VERSION, '5.3', '>=' ) ) {
+			$original_args['fallback_cb'] = '\\' . $original_args['fallback_cb'];
+		}
+
+    // Add global namespace prefix to check #41488.
```

Comment:
@senlin this is the problem with this PR - looks like a bad conflict resolution.  Lines 957-961 should be deleted.